smokes/e2e/pages/builder.ts

```diff
@@ -14,46 +15,50 @@ export class BuilderPage extends BasePage {
         this.forceName=forcename;
     }
 
-    goDefault() {
-        return browser.get('#/builders');
+    async goDefault() {
+        await browser.get('#/builders');
     }
 
-    go() {
-        browser.get('#/builders');
+    async go() {
+        await browser.get('#/builders');
+        await browser.wait(EC.urlContains('#/builders'),
+                           5000,
+                           "URL does not contain #/builders");
         const localBuilder = element.all(By.linkText(this.builder));
-        return localBuilder.click();
+        await localBuilder.click();
     }
 
-    goForce() {
-        this.go();
-        return element.all(By.buttonText(this.forceName)).first().click();
+    async goForce() {
+        await this.go();
+        await element.all(By.buttonText(this.forceName)).first().click();
     }
 
-    goBuild(buildRef) {
-        this.go();
-        return element.all(By.linkText(buildRef.toString())).click();
+    async goBuild(buildRef) {
```

Comment:
More of a TypeScript nit, what is `buildRef` and why do you need to `toString()` it. Having types will help everyone using this method to make sure they use it properly.

Reply:
Okay internal commentary: I understand this file was recently migrated over to TypeScript. So maybe the next PR?

Reply:
Yes, we can clean this up in a subsequent PR.  This code was converted from CoffeeScript -> JavaScript -> TypeScript, so there are lots of subsequent small code cleanups that can be done.

Reply:
Great! I'm excited to see what is next.